src/helpers/wallet.helper.js

```diff
@@ -44,6 +44,7 @@ export function initHathorLib(config) {
 export function getReadonlyWalletConfig({
   xpub,
   multisigData = null,
+  scanPolicy = null,
 }) {
   // Previous versions of the lib would have password and pin default as '123'
   // We currently need something to be defined, otherwise we get an error when starting the wallet
```

Comment:
[🚫 no entry] It is with the default value. Should we remove in this PR or another PR will solve it?

Reply:
This warning is only for the pin/password, the `scanPolicy` as `null` would use the default i.e. `gap-limit`